tests/Integration/Metadata/AuthorArchiveTest.php

```diff
@@ -68,7 +68,7 @@ public function test_author_archive(): void {
 		$this->assert_data_has_required_properties( $structured_data );
 
 		// The headline should be the category name.
-		self::assertEquals( 'Author - parsely', $structured_data['headline'] ?? null );
-		self::assertEquals( $author_posts_url, $structured_data['url'] ?? null );
+		self::assertSame( 'Author - parsely', $structured_data['headline'] ?? null );
+		self::assertSame( $author_posts_url, $structured_data['url'] ?? null );
```

Comment:
Aside: `?? null` is a bug waiting to happen :)

The value would either match `$structured_data['url']` or null and there should be separate tests if we're expecting both cases.

Not a huge issue, but something we should watch out for since this pattern is used several times in our tests.

Reply:
Noted. I'll look into this further once I get the chance.